deploy/playbooks/03_app.yml

```
@@ -41,10 +44,10 @@
       when: not env_file.stat.exists
 
     - name: Start docker compose to see if everything is running
-      shell: "docker compose up -d"
+      ansible.builtin.shell: "docker compose up -d"
 
     - name: Migrate on prod
-      shell: "make prod/migrate"
+      ansible.builtin.shell: "make prod/migrate"
 
     - name: Restart everything and finish
-      shell: "docker compose up -d"
+      ansible.builtin.shell: "docker compose up -d"
```

```suggestion
      ansible.builtin.command:
        chdir: {{ ansible_user_dir }}
        cmd: "docker compose up -d"
```
See above.
Does this really restart?